### PR TITLE
Use rollup instead of browserify

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 .DS_STORE
 node_modules
 dist
+dev/css/rando3D.css
+dev/app.*
+dev/index.html
```
Comment (bastyen) [inline .gitignore:6]: add dist ?

File: package.json [hunk 2]
```diff
@@ -24,30 +24,22 @@
...
+    "@rollup/plugin-commonjs": "^22.0.2",
+    "@rollup/plugin-html": "^0.2.4",
+    "@rollup/plugin-node-resolve": "^13.3.0",
+    "node-sass": "^7.0.1",
+    "rollup-plugin-livereload": "^2.0.5",
+    "rollup-plugin-scss": "^3.0.0",
+    "rollup-plugin-serve": "^2.0.1",
+    "rollup-plugin-sourcemaps": "^0.6.3",
+    "rollup-plugin-terser": "^7.0.2",
     "watchify": "^2.2.1"
```
Comment (bastyen) [inline package.json]: remove watchify ?

Comment (dtrucs) [reply package.json]: Yes indeed, i just removed it 👍

File: .gitignore (hunk 1 above)
Comment (dtrucs) [reply .gitignore:6]: line3?

File: dist/rando3D.css [hunk 3]
```diff
@@ -411,4 +207,3 @@
 
 .interface * {
   pointer-events: auto; }
-
```
Comment (bastyen) [inline dist/rando3D.css:414]: remove this file

Comment (dtrucs) [reply dist/rando3D.css:414]: Done 👍

**Review by bastyen — APPROVED** [review]

**Pull request merged**